blueprints/component-test/qunit-files/__root__/__testType__/__path__/__test__.ts

```diff
@@ -0,0 +1,40 @@
+<% if (testType == 'integration') { %>
```

Comment:
Let's just drop this conditional. Ember components technically support unit tests, but that's a catastrophically-bad antipattern, and Glimmer components don't support them at all. Just do rendering/integration tests!

Reply:
agree